src/veh_type.h

```diff
@@ -331,6 +331,9 @@ class vpart_info
         // Display order in vehicle interact display
         int list_order;
 
+        std::set<std::string> get_flags() const {
```

Comment:
```suggestion
        const std::set<std::string> &get_flags() const {
```
No need to copy the set.